src/components/ChatPopUp.tsx

```diff
@@ -124,6 +124,7 @@ export function ChatPopUp(props: ChatPopUpProps) {
           <ChatPanel
             {...props}
             customCssClasses={cssClasses.panelCssClasses}
+            isChatHidden={!showChat}
```

Comment:
this boolean prop feels a bit odd, can we instead render ChatPanel only when the button is clicked _for the first time_ and start the css transition as well?

Reply:
Good call - For some reason i thought it would always render, but seems to be simple enough to just another stateful flag which will prevent this.